model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/dycore_states.py

```diff
@@ -39,9 +40,9 @@ class DiagnosticStateNonHydro:
     ddt_w_adv_ntl2: fa.CellKField[float]
 
     # Analysis increments
```

Comment:
For the JW case those are initialized with `None`. So I patched the type annotation.